invenio_search/api.py

```diff
@@ -54,7 +54,8 @@ def query(self):
             tree = tree.accept(walker)
         return tree
 
-    def search(self, user_info=None, collection=None, **kwargs):
+    def search(self, user_info=None, collection=None, index="records*",
```

Comment:
can you keep `Query` class intact and amend only `Results` with default `index` argument? Later we make `Results` proxy configurable. Thanks